main.go

```diff
@@ -40,6 +41,26 @@ func main() {
 	buf := bufio.NewReader(os.Stdin)
 	var regArr []*regexp.Regexp
 
+	if *wordsParameter && !*fixedStringsParameter {
+		for i := range flagTest {
+			if !strings.HasPrefix(flagTest[i], "\\b") {
+				flagTest[i] = "\\b" + flagTest[i]
+			}
+			if !strings.HasSuffix(flagTest[i], "\\b") {
+				flagTest[i] = flagTest[i] + "\\b"
+			}
+		}
+	} else if *wordsParameter && *fixedStringsParameter {
+		for _, i := range flagTest {
+			re, regErr := regexp.Compile(regexp.QuoteMeta(i))
+			if regErr != nil {
+				fmt.Fprintln(os.Stderr, regErr)
+				os.Exit(1)
+			}
+			regArr = append(regArr, re)
+		}
```

Comment:
There's already a loop for regexp.Compile()ing. Any way not to duplicate that loop and to just regexp.QuoteMeta() the stuff in the new loop?